### DISCUSSION TITLE
Vercel domains .cv TLD support

### DISCUSSION THREAD

**Discussion by heyecs** [vercel/community#3832] [Feature Question] [answered]
### Question

I have a .cv domain I'd like to transfer to Vercel from another registrar. Does anyone know if Vercel supports transferring in .cv domains?

**Comment by amyegan** [comment, marked as answer]
Hey @heyecs. It is not currently possible to register a `.cv` domain with Vercel. But you can still use one with your project if you [add it as a custom domain](https://vercel.com/docs/projects/domains/add-a-domain) with A and CNAME records or by [using Vercel nameservers](https://vercel.com/docs/projects/domains/working-with-nameservers#).
